### ISSUE TITLE
conda-devenv install instruction is missing

### ISSUE THREAD

**Issue by fpetzold** [oursland/FreeCAD-Build-Notes#2]
Hi,

I am walking through the build instruction from scratch, step by step.

Several things do not work on my machine:

conda devenv is missing from the anaconda3 installation (I do not use home-brew but MacPorts and thus installed Anaconda3 with the GUI installer)

I installed devenv with

`conda install conda-devenv -c conda-forge`

But still, `conda devenv -f .vscode/environment.devenv.yml`

fails:

`File "/Users/fpe/compile/.vscode/environment.devenv.yml" does not exist.`

Next problem: 

`code` is an unknown command. I have to open VSCode from the Finder. Does that matter?

With the instructions as they were yesterday, I wasalble to compile with VSCode, but FreeCAD threw exceptions on start. I think some SDK was not installed, I remember instructions for installing this on another web page explaining how to build FreeCAD on macOS. Is it possible that something like that is missing as well?

**Comment by oursland** [comment]
@fpetzold 

The wrong branch was listed in the instructions.  `conda-devenv` is the new branch with the necessary configuration.  I have updated the instructions.

Running `code` from the command line should not be necessary, however I have found that sometimes the VS Code's environment variables when launched from Finder are inconsistent.  Sometimes the environment is fairly bare, and other times it contains environment as if it was launched from a login shell with the PATH updated and conda configured.
